sourceclear.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
I'd put this script under Scripts: https://github.com/optimizely/objective-c-sdk/tree/master/Scripts